libs/blocks/aside/aside.js

```diff
@@ -84,6 +91,25 @@ function decorateVideo(container) {
   container.classList.add('has-video');
 }
 
+function formatPromoButton(el) {
+  if (!(el.classList.contains('promobar') && el.classList.contains('popup'))) return;
+  const actionArea = el.querySelectorAll('.action-area');
+  [...actionArea].forEach((aa) => {
```

Comment:
You can use `[...el.querySelectorAll('.action-area')].forEach` directly. Same goes for line 99

Reply:
@narcis-radu  I have made the suggested change.